### DISCUSSION TITLE
Yeees… but no

### DISCUSSION THREAD

**Discussion by noctyris** [community/community#67495] [Other Feature Feedback, Questions, & Ideas]
The app is good 👍🏼 but the problem 👎🏼 is that we can't create **repostery**, **pull request**, change of **branch** or other _(or I'm just stupid 'cause it's possible 😂)_. If we can but I don't know how to do it, indicate how to do all this please. 🥺

Thank you and goodbye

**Comment by KostaD02** [comment]
Hi there! It sounds like you're enjoying the app, which is great to hear! Don't worry, you're not alone in wondering how to perform certain actions. Let me guide you through the process:

Creating a Repository:

- On GitHub, navigate to your profile.
- Click on the "+ New" button on the right-hand side.
- Select "Repository" from the dropdown menu.
- Fill in the necessary details and click "Create repository."

Creating a Pull Request:

- In your repository, click on the "Pull Requests" tab.
- Click the green "New pull request" button.
- Select the base and compare branches, and click "Create pull request."
- Add a title and description, then click "Create pull request" again.

Changing Branches:

- On the main page of your repository, you'll find a dropdown menu that displays the current branch. Click on it and select the branch you want to switch to.

You can use git or github desktop application for this processes.

Happy coding!

**Reply by noctyris** [reply]
Hello, does your technic work in the mobile app (iOS) ?
I haven't the buttons you said. 
That's a bit weird else

**Reply by KostaD02** [reply]
I have never tried to create a repository or pull request from my phone. There may be a way to achieve the same result, but perhaps with a different approach. For a much better experience, you should try it from a PC. However, if you insist on using a phone (from an app, I don't know), open a browser and try to request the desktop site.

**Reply by noctyris** [reply]
Ok, I'll contact GitHub to have an answer 😉
Thanks you

**Reply by KostaD02** [reply]
You're welcome
